ADAL/src/ADRequestParameters.m

```diff
@@ -27,9 +27,50 @@
 #import "MSIDAccountIdentifier.h"
 #import "NSString+MSIDExtensions.h"
 #import "MSIDAuthorityFactory.h"
+#import "MSIDConstants.h"
 
 @implementation ADRequestParameters
 
+@synthesize authority = _authority;
+@synthesize resource = _resource;
+@synthesize clientId = _clientId;
+@synthesize redirectUri = _redirectUri;
+@synthesize identifier = _identifier;
+@synthesize extendedLifetime = _extendedLifetime;
+@synthesize forceRefresh = _forceRefresh;
+@synthesize correlationId = _correlationId;
+@synthesize telemetryRequestId = _telemetryRequestId;
```

Comment:
do we need to synthesize them manually? :)

Reply:
Fixed, it got added as part of the merge :)